Game_Pencil_Engine_IDE.cbp

```diff
@@ -0,0 +1,542 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Ok, this time it really is the absolutely last thing. Replace this entire Game_Pencil_Engine.cbp file with:
https://pastebin.com/pAV3uNFQ